docs/make.jl

```diff
@@ -0,0 +1,8 @@
+using Documenter, StatsModels
+
+makedocs()
```

Comment:
Might be worth considering the new HTML format supported by Documenter, https://juliadocs.github.io/Documenter.jl/latest/man/guide.html#Output-formats-1, since it's already being used by other JuliaStats packages (`DataFrames`) and might be nice to have a consistent documentation style across the entire organisation.